cadl-tests/src/test/java/com/models/property/optional/DurationOperationClientTest.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.models.property.optional;
+
+import com.models.property.optional.models.DurationProperty;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+@Disabled("cadl-ranch definition error")
```

Comment:
disabled these tests due the a bug in cadl-ranch, I've created a fix pr in cadl-ranch: https://github.com/Azure/cadl-ranch/pull/113